docs/sources/alerting/set-up/provision-alerting-resources/terraform-provisioning/index.md

```diff
@@ -1,367 +0,0 @@
----
```

Comment:
I wonder why we delete this article. It's still relevant and very informative. We can amend it by mentioning export in the sections `Provision contact points and template` and below as well as modify export in 4th section.

Reply:
? confused because during our meeting i thought you had said it is only for cloud. can you confirm that this topic should be in both cloud and oss? and also that file provisioning should not be in cloud? thanks

Reply:
Apologies for not being clear. I meant that API provisioning and Terraform as the only ready-to-use client we offer, is the only available provisioning method in Cloud. The second method - file provisioning, is not available in Cloud because we do not offer access to the file system of the Grafana instance.
Therefore:
Cloud: API (Terraform)
On-premise (OSS, GE): API (Terraform) and File provisioning